lib/util/prefs_util.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+class PrefsUtil {
+  static Future<String> getUserId() async {
+    final prefs = await SharedPreferences.getInstance();
+    return prefs.getString('user_id') ?? "";
```

Comment:
It is not better to initialize as NULL?

Reply:
Method isNotEmpty is called on returned value